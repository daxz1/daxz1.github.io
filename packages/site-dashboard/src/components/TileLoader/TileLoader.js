import React, { memo } from "react";

export const TileLoader = memo(() => (
  <div className="tile">
    <div className="loader" />
  </div>
));
