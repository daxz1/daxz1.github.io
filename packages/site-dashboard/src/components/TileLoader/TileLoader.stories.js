import React from "react";
import { TileLoader } from "./TileLoader";

export default {
  title: 'components/TileLoader',
  component: TileLoader
};

export const StandardView = () => <TileLoader />;

StandardView.storyName = 'Default View';