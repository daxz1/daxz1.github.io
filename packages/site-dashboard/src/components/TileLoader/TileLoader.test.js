/*global describe, test*/

import React from "react";
import { create } from "react-test-renderer";
import { TileLoader } from "./TileLoader";

describe("TileLoader Component", () => {
  test("Matches the snapshot", () => {
    const Application = create(<TileLoader />);
    expect(Application.toJSON()).toMatchSnapshot();
  });
});
