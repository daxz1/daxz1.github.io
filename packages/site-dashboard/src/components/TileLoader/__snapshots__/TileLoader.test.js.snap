// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TileLoader Component Matches the snapshot 1`] = `
<div
  className="tile"
>
  <div
    className="loader"
  />
</div>
`;
